.github/workflows/continuous-integration.yaml

```diff
@@ -31,20 +31,12 @@ jobs:
     strategy:
       matrix:
         php:
-          - '7.2'
-          - '7.3'
-          - '7.4'
           - '8.0'
           - '8.1'
         dependencies:
           - 'highest'
         elasticsearch:
           - '7.15.2'
-        include:
-          # Test with the lowest set of dependencies
-          - dependencies: 'lowest'
-            php: '7.2'
-            elasticsearch: '7.15.2'
```

Comment:
I think this should be modified instead of removed, the idea of this job is to run the tests with the lowest set of dependencies.

Let's say we require `elasticsearch/elasticsearch: ^8.0` package, this job would use `8.0` to run the tests, otherwise we would only test with the latest release (right now `8.5.2`)

Reply:
Yeah the idea is clear but currently (in this PR) we don't have lower dependencies (except probably psr/log). We could reintroduce testing against lower dependencies soon when `elasticsearch/elasticsearch` will be updated and we decide which minimum version of it we will use

Reply:
Maybe a compromise here is to comment it out for now and leave a comment. I'm a bit worried we might forget to readd it.

Reply:
reverted

Reply:
thx